artifacts/openapi.yaml

```diff
@@ -928,6 +951,380 @@ components:
           format: mac
           default: 00:00:00:00:00:00
           x-field-uid: 3
+    Device.Dhcpv4client:
+      description: |-
+        Under Review: Information TBD
+
+        Configuration for emulated DHCPv4 Client on a single Interface. https://www.rfc-editor.org/rfc/rfc2131.html
+      type: object
+      x-status:
+        status: under_review
+        information: Information TBD
+      required:
+      - name
+      properties:
+        name:
+          x-field-uid: 1
+          description: |-
+            Globally unique name of an object. It also serves as the primary key for arrays of objects.
+          type: string
+          pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
+          x-unique: global
+        selecting_server:
+          description: |-
+            The client receives one or more DHCPOFFER messages from one or more servers. The client may choose to wait for multiple responses. The client chooses one server from which to request configuration parameters, based on the configuration parameters offered in the DHCPOFFER messages. use_first_server: if selected, the subnet accepts the IP addresses offered by the first server to respond with an offer of IP addresses. use_server_address: The address of the DHCP server from which the subnet will accept IP addresses. server address to 0.0.0.0, implementation will use the services of the first DHCP server that responds to the DHCP Lease Request message.
+          type: string
+          default: first_server
+          x-field-uid: 2
+          x-enum:
+            first_server:
+              x-field-uid: 1
+            server_address:
+              x-field-uid: 2
+          enum:
+          - first_server
+          - server_address
+        first_server:
+          type: string
+          format: ipv4
+          default: 0.0.0.0
+          x-field-uid: 3
+        server_address:
+          type: string
+          format: ipv4
+          default: 0.0.0.0
+          x-field-uid: 4
+        broadcast:
+          description: |-
+            If the broadcast bit is set, then the server and relay agent broadcast DHCPOFFER and DHCPACK messages.
```

Comment:
The description should be: If the broadcast bit it set, client asks the server or the relay agent to broadcast...

There's a difference in the two meanings. Please check IxN or RFC.